crates/wdk-sys/src/types.rs

```diff
@@ -10,8 +10,20 @@
 #[allow(clippy::cast_possible_truncation)]
 #[allow(clippy::cognitive_complexity)]
 #[allow(clippy::default_trait_access)]
-#[rustversion::attr(before(2023-09-10), allow(clippy::incorrect_clone_impl_on_copy_type))]
-#[rustversion::attr(since(2023-09-10), allow(clippy::non_canonical_clone_impl))]
+#[rustversion::attr(
```

Comment:
fix in this attribute is due to upstream rustversion issue: https://github.com/dtolnay/rustversion/issues/42

fix here was tested on the stables, betas, and nightlies bordering the listed versions